src/store/plugins/geolocation-management.plugin.js

```diff
@@ -101,20 +102,23 @@ const handlePositionError = (error, store, state, options = {}) => {
                 ...dispatcher,
             })
             break
-        case error.TIMEOUT:
+        case GeolocationPositionError.TIMEOUT:
             store.dispatch('setGeolocation', { active: false, ...dispatcher })
             store.dispatch('addErrors', {
                 errors: [new ErrorMessage('geoloc_time_out', null)],
                 ...dispatcher,
             })
             break
         default:
-            if (IS_TESTING_WITH_CYPRESS && error.code === error.POSITION_UNAVAILABLE) {
-                // edge case for e2e testing, if we are testing with Cypress and we receive a POSITION_UNAVAILABLE
-                // we don't raise an error as it's "normal" in Electron to have this error raised (this API doesn't work
-                // on Electron embedded in Cypress : no Geolocation hardware detected, etc...)
-                // the position will be returned by a mocked up function by Cypress we can ignore this error
-                // we do nothing...
+            if (
+                IS_TESTING_WITH_CYPRESS &&
```

Comment:
we could get rid if this `IS_TESTING_WITH_CYPRESS` here, and apply the code bellow anytime this error code is found? (could change it to another case in the switch if it's the case)

Reply:
Or if the goal is to disable the retry function, we could add some `IS_TESTING_WITH_CYPRESS` case in the code below, such as
```javascript
errorCount += IS_TESTING_WITH_CYPRESS ? 3 : 1
```

Reply:
I am not sure about the original intention, but it seems it's used to prevent trying more than once when doing test. I have updated as you suggested.